src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -490,11 +491,16 @@ export default class ListStore {
 
         log.info('List loads "' + this.resourceKey + '" data with the following options:', options);
 
-        this.loadingStrategy.load(
+        if (this.pendingRequest instanceof RequestPromise) {
```

Comment:
wouldnt it be more intuitive to check if it is set/not undefined? 🙈

Reply:
Actually I thought that it makes more sense this way... I even thought about a `CancelablePromise` or something like that, but the issue is that only requests can be cancelled.

Reply:
Realized that because of some type changes I don't even need the `instanceof` anymore, therefore I removed it :slightly_smiling_face: